apps/backend/src/routes/sendEmail.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const { getTransporter } = require('../utils/getTransporter');
+const { withEmailTemplate } = require('../utils/withEmailTemplate');
+
+const router = express.Router();
+
+router.post('/', async (req, res) => {
+  try {
+    const emailDataArray = req.body;
+
+    const emailInfoArray = [];
```

Comment:
Add some zod validation to make sure that we are getting an array with the keys that we need.

If not return an error.

Please look at the other routes in this backend to see how it is used